pkg/irreparable/irreparable.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package irreparable
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+
+	"storj.io/storj/internal/migrate"
+	dbx "storj.io/storj/pkg/irreparable/dbx"
+)
+
+var (
+	mon = monkit.Package()
```

Comment:
No need for monitoring. Although, not quite sure.

Reply:
Done